docs/products/kafka/howto/provectus-kafka-ui.rst

```diff
@@ -0,0 +1,50 @@
+Use Provectus® UI for Apache Kafka® with Aiven for Apache Kafka®
+================================================================
+
+`Provectus® UI for Apache Kafka® <https://github.com/provectus/kafka-ui>`_ is a popular Open-Source web GUI for Apache Kafka® management that allows you to monitor and manage Apache Kafka® clusters.
```

Comment:
Usually you only need the "r" symbol on the first and most appearance fo the word.